### PR TITLE
feat: new tokens for link component

### PR DESCRIPTION
**Pull request by daine**
# Summary | Résumé
Tokens for the new Link component

### CODE REVIEW

File: tokens/components/link/tokens.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "link": {
+    "font": {
+      "value": {
+        "fontFamily": "{fontFamilies.body}",
+        "fontWeight": "{fontWeights.medium}",
```
Comment (melaniebmn) [inline tokens/components/link/tokens.json:6]: Can you please change the `fontWeight` from `medium` to `regular` to match the design and general text style?

File: tokens/components/link/tokens.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      },
+      "type": "typography"
+    },
+    "focus": {
+      "text": {
+        "value": "{focus.text.value}",
+        "type": "color"
+      }
+    },
+    "hover": {
```
Comment (melaniebmn) [inline tokens/components/link/tokens.json:18]: Let's remove this token and use the global link hover token instead to not duplicate tokens and keep it a bit simpler :)

File: tokens/components/link/tokens.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "font": {
+      "value": {
+        "fontFamily": "{fontFamilies.body}",
+        "fontWeight": "{fontWeights.medium}",
+        "lineHeight": "{lineHeights.text}",
+        "fontSize": "{fontSizes.text}"
+      },
+      "type": "typography"
+    },
+    "focus": {
```
Comment (melaniebmn) [inline tokens/components/link/tokens.json:12]: Can you please add the other focus tokens for the link that are used for the focus style here:
https://github.com/cds-snc/gcds-components/pull/305/files#diff-7d564b71f770be346b3d550da929935fdb9ede207e882510069ef7d081902fb8R18

You can copy and paste them from the button tokens here:
https://github.com/cds-snc/gcds-tokens/blob/main/tokens/components/button/tokens.json#L137
https://github.com/cds-snc/gcds-tokens/blob/main/tokens/components/button/tokens.json#L3

File: tokens/components/link/tokens.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        "type": "color"
+      }
+    },
+    "small": {
+      "font": {
+        "value": {
+          "fontFamily": "{fontFamilies.body}",
+          "fontWeight": "{fontWeights.regular}",
+          "lineHeight": "{lineHeights.text}",
+          "fontSize": "80%"
```
Comment (melaniebmn) [inline tokens/components/link/tokens.json:30]: And one more: can you check with Nick please if we will support smaller font sizes for this component? If yes, we can use `caption` for `lineHeight` and `fontSize`.

**Review by melaniebmn — CHANGES_REQUESTED** [review]
Added a few comments, let me know if you have any questions about these!

**Comment by daine** [comment]
@melaniebmn verified with @nmakuch that:
- There would be no extra styling for active state for links
- `caption` size is available as a smaller font for links

**Review by melaniebmn — CHANGES_REQUESTED** [review]
Awesome, thanks for double-checking about the size and active state!

The tokens look good! Can you please update the package version to `1.8.0`? After that, the PR should be good to go.

**Review by melaniebmn — APPROVED** [review]
LGTM :)

**Pull request merged**